pkg/reconciler/taskrun/resources/apply_test.go

```diff
@@ -839,11 +839,15 @@ func TestContext(t *testing.T) {
 			}},
 		},
 	}, {
-		description: "context taskRunName replacement with no defined taskRun in spec container",
```

Comment:
Have we lost a bit of cover here or was it intentional to remove the test where the taskRun has no name defined?